TestModels/SharedMakefile.mk

```diff
@@ -185,28 +246,31 @@ transpile_dependencies:
 # StandardLibrary is filtered out from dependent-model patsubst list;
 #   Its model is contained in $(LIBRARY_ROOT)/model, not $(LIBRARY_ROOT)/../StandardLibrary/Model.
 _polymorph:
-	@: $(if ${CODEGEN_CLI_ROOT},,$(error You must pass the path CODEGEN_CLI_ROOT: CODEGEN_CLI_ROOT=/path/to/smithy-dafny/codegen/smithy-dafny-codegen-cli));
```

Comment:
I like this option... I will not block on it's removal, 
but I see it's value if I am working on Smithy-Dafny itself,
and want to test it against the MPL/Gazelle with out pushing and pulling it.

Reply:
I'm not removing it, it just has a default value now, so it's not required but you can override it! We already made this change in the smithy-dafny version: https://github.com/smithy-lang/smithy-dafny/blob/main-1.x/TestModels/SharedMakefile.mk#L33